src/plugins/matrixops/sum_operation.cpp

```diff
@@ -32,7 +32,7 @@ namespace phylanx { namespace execution_tree { namespace primitives
     ///////////////////////////////////////////////////////////////////////////
     match_pattern_type const sum_operation::match_data =
     {
-        hpx::util::make_tuple("sum_operation",
+        hpx::util::make_tuple("sum",
```

Comment:
This requires changing the name here as well: https://github.com/STEllAR-GROUP/phylanx/blob/master/phylanx/plugins/matrixops/sum_operation.hpp#L77

Reply:
This comment doesn't make sense. I don't have the old name anywhere.

Reply:
Not anymore, yes.